utils/byobu/Makefile

```diff
@@ -37,6 +37,7 @@ define Package/byobu
 $(call Package/byobu/Default)
   TITLE:=Text-based window manager and terminal multiplexer
   DEPENDS:=+python3-light +python3-newt
+  select tmxu if !screen
```

Comment:
This has to be inside of a config section (`define Package/byobu/config`) to work.

But more importantly, this is a directive for the build system; it doesn't translate into dependency information for opkg, which is what you want (and why this isn't specified in the makefile already).